Doc/library/tkinter.rst

```diff
@@ -67,13 +67,33 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1, sync=0, use=None)
```

Comment:
It is better to use booleans as default values: `useTk=True, sync=False`.

Reply:
Yes, agreed.  Should tkinter.__init__.py be changed too or would that be noise?

Reply:
This is a pure documentation issue. It will be backported to other branches. It isn't worth to backport the code changes.